import { AppException } from '../app.exception';

export class ConsultaNaoEncontradoException extends AppException {
  constructor() {
    super('Consulta n√£o encontrado!', 404);
  }
}